doc/source/georeferencing.md

```diff
@@ -0,0 +1,214 @@
+{attr}`~geoutils.Raster.shape` made up of its {attr}`~geoutils.Raster.height` and {attr}`~geoutils.Raster.width`.
+
+```{code-cell} ipython3
+# Get raster transform and shape
+print(rast.transform)
+print(rast.shape)
+```
+
+(pixel-interpretation)=
+### Pixel interpretation (only for rasters)
+
+A largely overlooked aspect of a raster's georeferencing is the pixel interpretation stored in the 
+[AREA_OR_POINT metadata](https://gdal.org/user/raster_data_model.html#metadata). 
+Pixels can be interpreted either as **"Area"** (the most common) where **the value represents a sampling over the region 
+of the pixel (and typically refers to the upper-left corner coordinate)**, or as **"Point"**
+where **the value relates to a point sample (and typically refers to the center of the pixel)**, used often for DEMs. 
+
+Pixel interpretation is stored as a string in the {attr}`geoutils.Raster.area_or_point` attribute. 
+
+```{code-cell} ipython3
+# Get pixel interpretation of raster
+rast.area_or_point
+```
+
+Although this interpretation is not intended to influence georeferencing, it **can influence sub-pixel coordinate
+interpretation during analysis**, especially for raster–vector–point interfacing operations such as point interpolation, 
+or re-gridding, and might also be a problem if defined differently when comparing two rasters.
+
+```{important}
+By default, **pixel interpretation can induce a half-pixel shift during raster–point interfacing** 
+(mirroring [GDAL's default ground-control point behaviour](https://trac.osgeo.org/gdal/wiki/rfc33_gtiff_pixelispoint)), 
+but only **raises a warning for raster–raster operations**.
+
+This behaviour can be modified at the package-level by using [GeoUtils' configuration parameters]() 
+`shift_area_or_point` and `warns_area_or_point`.
+```
+
+## Manipulation
+
+Several functionalities are available to facilitate the manipulation of the georeferencing.
+
+### Getting projected bounds and footprints
+
+Retrieving projected bounds or footprints in any CRS is possible using directly {func}`~geoutils.Raster.get_bounds_projected` 
+and {func}`~geoutils.Raster.get_footprint_projected`.
+
+```{code-cell} ipython3
+# Get footprint of larger buffered vector in polar stereo CRS (to show deformations)
+vect.buffer_metric(10**6).get_footprint_projected(3995).plot()
+```
+
+### Getting a metric CRS
+
+A local metric coordinate system can be estimated for both {class}`Rasters<geoutils.Raster>` and {class}`Vectors<geoutils.Vector>` through the
+{func}`~geoutils.Raster.get_metric_crs` function.
+
+The metric system returned can be either "universal" (zone of the Universal Transverse Mercator or Universal Polar Stereographic system), or "custom"
+(Mercator or Polar projection centered on the {class}`Raster<geoutils.Raster>` or {class}`Vector<geoutils.Vector>`).
+
+```{code-cell} ipython3
+# Get local metric CRS
+print(rast.get_metric_crs())
+```
+
+### Re-set georeferencing metadata
+
+To add?
```

Comment:
Good idea ! Although this is of course easily done with `Raster.from_array`. A simpler option is to have a single function `set_georef` that allows overwriting any of these.